app/components/vistk-scatterplot.js

```diff
@@ -50,6 +50,60 @@ export default Ember.Component.extend({
           fill: (d) => { return d.color ? d.color : '#ccc1b9'; }
         }, {
           var_mark: '__highlighted',
+          type: d3.scale.ordinal().domain([true, false]).range(['line_horizontal', 'none']),
+          offset_right: function(d, i, vars) {
+              return vars.x_scale[0]['func'].range()[1] - vars.x_scale[0]['func'](d[vars.var_x]) + vars.r_scale(d[vars.var_r]);
+          }
+        }, {
+          var_mark: '__highlighted',
+          type: d3.scale.ordinal().domain([true, false]).range(['line_vertical', 'none']),
+          offset_top: function(d, i, vars) {
+              return vars.r_scale(d[vars.var_r]);
+          }
+        }, {
+          var_mark: '__highlighted',
+          type: d3.scale.ordinal().domain([true, false]).range(['rect', 'none']),
+          translate: function(d, i, vars) {
+            return [-vars.x_scale[0]['func'](d[vars.var_x]) - 20, -10];
+          },
+          height: 25,
+          width: 50,
+          stroke: 'black',
+          stroke_width: '1.5px',
+          fill: function() { return 'white'; }
+        }, {
+          var_mark: '__highlighted',
+          type: d3.scale.ordinal().domain([true, false]).range(['text', 'none']),
+          translate: function(d, i, vars) {
+            return [-vars.x_scale[0]['func'](d[vars.var_x]) + 25, 0];
+          },
+          text_anchor: 'end',
+          text: function(d, i, vars) {
+            return format(d[vars.var_y]);
```

Comment:
tooltips are breaking for me, var_y is undefined when I'm on the potential products or industries for Bogota

http://ci-deploys.cid-labs.com/cid-harvard/atlas-subnational-frontend/feature/scatter-axis-values/435/#/location/3/source/industries/visualization/scatter/opportunity?endDate=2013&startDate=2013